client/src/io/fs.rs

```diff
@@ -1,7 +1,7 @@
 //! Deals with all filesystem specific details
 
-use crate::{consts, profiles::Profile, Result};
-use std::path::PathBuf;
+use crate::{consts, error::ClientError, profiles::Profile, Result};
```

Comment:
I guess I should run clippy cross platform as well. Anyway this will be an unused import on windows and should be cfg-gated.